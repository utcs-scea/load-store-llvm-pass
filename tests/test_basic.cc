#include <stdio.h>
#include <stdint.h>

uint64_t hello = 5;

int main() {
  uint64_t* bravo = &hello;
  printf("hello is == %llu.\n", *bravo);
}

