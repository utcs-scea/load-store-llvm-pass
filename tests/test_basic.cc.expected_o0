   >> globalify() invoked
   >> globalify() invoked
   >> __pando__replace_store_ptr() invoked
   >> check_if_global() invoked
   >> deglobalify() invoked
   >> __pando__replace_load_ptr() invoked
   >> check_if_global() invoked
   >> deglobalify() invoked
   >> __pando__replace_load_int64() invoked
   >> check_if_global() invoked
   >> deglobalify() invoked
   >> globalify() invoked
   >> deglobalify() invoked
hello is == 5.
