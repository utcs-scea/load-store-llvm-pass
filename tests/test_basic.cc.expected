   >> globalify() invoked
   >> __pando__replace_load_int64() invoked
   >> check_if_global() invoked
   >> deglobalify() invoked
   >> globalify() invoked
   >> deglobalify() invoked
hello is == 5.
